test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -156,13 +151,12 @@ class CloudFoundryDeployTest extends BasePiperTest {
             cfAppName: 'testAppName',
             cfManifest: 'test.yml'
         ])
-
-        assertTrue(jscr.shell[1].contains('cf login -u "test_cf" -p \'********\' -a https://customApi -o "testOrg" -s "testSpace"'))
+        // asserts
```

Comment:
unrelated to scope of PR, only another way of asserting --> should be done in a dedicate PR.

Reply:
needed to change assertion because of `assertTrue(jscr.shell[0].contains...`